protocol/specification/descriptive.rst

```diff
@@ -126,44 +126,108 @@ Optional Module Properties
 
     Related issue: :issue:`008 Groups and Hierarchy`
 
+.. _module-meaning:
+
 ``"meaning"``
-    A tuple, with the following two elements:
+   A JSON object with data regarding the module meaning. It provides metadata that is useful for interpreting measurement data in an automatic fashion. It can have the keys ``function``, ``importance``, ``belongs_to``, ``link`` and ``key``, all of which are optional, with some restrictions. A meaning property can also be added on the :ref:`parameter-meaning <paremeter level>`.
```

Comment:
type: "parameter level" instead of "parameter level"

Reply:
I am not sure what you mean here did you want to write "parameter-level" ?

I could also rewrite it to:
A meaning property can also be added as an optional parameter property.

Reply:
sorry, my comment is confusing - its just a typo. you typed <paremeter level> instead of <parameter level>

Reply:
again - its just a typo. you typed "paremeter" instead of "paramter"